sschipke/palette-producer
# Ss redux testing

## What's this PR do?
- Adds tests for all components, reducers, and actions
### Where should the reviewer start?
- All the test files, especially reducers and actions tests
### How should this be manually tested?
- run `nmp test` or `npm test --  --coverage --watchAll=false` to see the test coverage
